public/logos/pocket.svg

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is the wordmark logo. Needs to be replaced with Pocket's circular icon/badge version.

https://assets.website-files.com/609e7a6f2ec5c05d866ed6d3/60a7d6075adffd5ce48ea5da_POKT_Symbol_Color.svg